src/PrestaShopBundle/Twig/LayoutExtension.php

```diff
@@ -74,22 +76,29 @@ public function getGlobals(): array
 
         try {
             $defaultCurrency = $this->context->getEmployeeCurrency() ?: $this->currencyDataProvider->getDefaultCurrency();
-        } catch (\Exception $e) {
+        } catch (\Exception) {
             $defaultCurrency = null;
         }
         try {
             $rootUrl = $this->context->getRootUrl();
-        } catch (\Exception $e) {
+        } catch (\Exception) {
             $rootUrl = null;
         }
 
+        try {
+            $useSymfonyLayout = $this->featureFlagStateChecker->isEnabled('symfony_layout');
+        } catch (\Exception) {
+            $useSymfonyLayout = false;
+        }
+
         return [
             'theme' => $this->context->getContext()->shop->theme,
             'default_currency' => $defaultCurrency,
             'default_currency_symbol' => $defaultCurrency instanceof Currency ? $defaultCurrency->getSymbol() : null,
             'root_url' => $rootUrl,
             'js_translatable' => [],
             'rtl_suffix' => $this->context->getContext()->language->is_rtl ? '_rtl' : '',
+            'use_symfony_layout' => $useSymfonyLayout,
```

Comment:
Actually, since we have no system that prevents the `LayoutBuilder` from being built and injected as a global variable this could probably be moved inside it and initialized in the constructor so that we can access this via `ps.isUsingSymfonyLayout` for example

Since the object is always there (and will likely stay there once symfony layout is by default), we might as well handle this condition there as well